tests/test_utils.py

```diff
@@ -179,11 +180,11 @@ def test_get_ssl_cert_file():
             {"https": "http://127.0.0.1", "http": "https://127.0.0.1"},
         ),
         (
-            {"all_proxy": "http://127.0.0.1", "ALL_PROXY": "https://1.1.1.1"},
+            {"all_proxy": "http://127.0.0.1", "_ALL_PROXY": "https://1.1.1.1"},
             {"all": "http://127.0.0.1"},
         ),
         (
-            {"https_proxy": "http://127.0.0.1", "HTTPS_PROXY": "https://1.1.1.1"},
+            {"https_proxy": "http://127.0.0.1", "_HTTPS_PROXY": "https://1.1.1.1"},
```

Comment:
Same with `_ALL_PROXY` - it's not obvious what's going on here?

Reply:
due to same reason.